### PR TITLE
Add titles to CMS pages, addressing #536

### PR DESCRIPTION
**Pull request by bkettle**
These changes define the title strings for all pages in the CMS in order to address #536, and also change the format of the title to `<contextual title> | TT CMS` in order to clearly show the difference between the frontend and backend.

### CODE REVIEW

File: app/controllers/ads_controller.rb [hunk 1]
```diff
@@ -7,20 +7,24 @@ class AdsController < ApplicationController
   respond_to :html
 
   def index
+    @title = 'Ads'
```
Comment (CyanoKobalamyne) [inline app/controllers/ads_controller.rb:10]: Can you change all the strings to use double quotes for consistency?

File: app/views/frontend/issue_index.haml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-- Issue.reorder(:volume).last.volume.downto(1) do |i|
+- Issue.published.reorder(:volume).last.volume.downto(1) do |i|
```
Comment (CyanoKobalamyne) [inline app/views/frontend/issue_index.haml:1]: These changes seem to be unrelated to this pull request?

**Review by CyanoKobalamyne — CHANGES_REQUESTED** [review]

File: app/views/frontend/issue_index.haml (hunk 2 above)
Comment (bkettle) [reply app/views/frontend/issue_index.haml]: Oops, these were from something else that I was trying. Fixed now.

File: app/controllers/ads_controller.rb (hunk 1 above)
Comment (bkettle) [reply app/controllers/ads_controller.rb]: Good point, done!

**Review by CyanoKobalamyne — APPROVED** [review]
